src/error_messages/information_messages.h

```diff
@@ -58,6 +58,7 @@
 #define FIM_FILE_SIZE_LIMIT_DISABLED        "(6042): File size limit disabled."
 #define FIM_DISK_QUOTA_LIMIT_DISABLED       "(6043): Disk quota limit disabled."
 #define FIM_NO_DIFF_REGISTRY                "(6044): Option nodiff enabled for %s '%s'."
+#define FIM_AUDIT_CREATED_RULE_FILE         "(6045): Created audit rule file, due to audit immutable mode, rules will be loaded in the next reboot"
```

Comment:
`FIM_AUDIT_CREATED_RULE_FILE` is used in a debug message, it should go in `debug_messages.h` instead of `information_messages.h`, other than that the PR is working fine.

Reply:
I'm going to turn it into an info message, I think it's a piece of information that will only appear in Wazuh restarts, it makes sense that it's a minfo.